src/cairoWriter/writers/variableDeclarationStatementWriter.ts

```diff
@@ -68,17 +69,17 @@ export class VariableDeclarationStatementWriter extends CairoASTNodeWriter {
         return [writer.write(getDeclarationForId(id))];
       }
     });
+    const solTyp = safeGetNodeType(node.vInitialValue, this.ast.inference);
+    const cairoTyp = CairoType.fromSol(solTyp, this.ast).toString();
+    let namePortion: string;
     if (declarations.length > 1) {
-      return [
-        endent`${documentation}
-          let (${declarations.map((decl) => `mut ${decl}`).join(', ')}) = ${writer.write(
-          node.vInitialValue,
-        )};`,
-      ];
+      namePortion = `(${declarations.map((decl) => `mut ${decl}`).join(', ')})`;
+    } else {
+      namePortion = `mut ${declarations[0]}`;
```

Comment:
I would prefer to handle this with a ternary expression. It would look something like:
```
const namePortion: string = (declarations.length > 1) 
      ? `(${declarations.map((decl) => `mut ${decl}`).join(', ')})`
      : `mut ${declarations[0]}`;
```
This is only a style suggestion, of course, it's your call.

Reply:
While we're at it, it'd probably be better to extract ```(decl) => `mut ${decl}` ```. There's too many nested backticks and brackets

Reply:
+1 seems like a clearer option to me in this case